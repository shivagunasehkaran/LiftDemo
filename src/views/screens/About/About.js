import React, {Component} from 'react';
import {StyleSheet, Text, View} from 'react-native';
import theme from '../../../themes';

class About extends Component {
    static navigationOptions = ({navigation}) => ({
        title: ('About'),
        headerStyle: {
            backgroundColor: theme.colors.buttonColor,
        },
    });

    render() {
        return (
            <View>
                <Text style={styles.detailText}>
                    {('The sample Lift App developed for demo purpose and This App is only for showing the lift logic and UI designs.')}
                </Text>
                <Text style={styles.addressText}>
                    {('Shiva Gunasehkaran')}
                </Text>
                <Text style={styles.phoneNumberText}>
                    {('Contact me at +91 9500936053')}
                </Text>
            </View>
        );
    }
}

const styles = StyleSheet.create({
    detailText: {
        color: theme.colors.black,
        padding: 20,
        textAlign: 'center',
        fontSize: 18,
        marginTop: 200,
    },
    addressText: {
        fontWeight: 'bold',
        textAlign: 'center',
        marginTop: 100,
        fontSize: 22,
    },
    phoneNumberText: {
        fontWeight: 'bold',
        textAlign: 'center',
        marginTop: 10,
        fontSize: 15,
    },
});

export default About;
