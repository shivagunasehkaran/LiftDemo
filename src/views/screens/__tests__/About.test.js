import React from 'react';
import renderer from 'react-test-renderer';
import About from '../About/About';

test('renders correctly about screen', () => {
    const tree = renderer.create(<About />).toJSON();
    expect(tree).toMatchSnapshot();
});
