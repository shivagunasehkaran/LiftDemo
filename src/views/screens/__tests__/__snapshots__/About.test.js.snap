// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly about screen 1`] = `
<View>
  <Text
    style={
      Object {
        "color": "#000000",
        "fontSize": 18,
        "marginTop": 200,
        "padding": 20,
        "textAlign": "center",
      }
    }
  >
    The sample Lift App developed for demo purpose and This App is only for showing the lift logic and UI designs.
  </Text>
  <Text
    style={
      Object {
        "fontSize": 22,
        "fontWeight": "bold",
        "marginTop": 100,
        "textAlign": "center",
      }
    }
  >
    Shiva Gunasehkaran
  </Text>
  <Text
    style={
      Object {
        "fontSize": 15,
        "fontWeight": "bold",
        "marginTop": 10,
        "textAlign": "center",
      }
    }
  >
    Contact me at +91 9500936053
  </Text>
</View>
`;
